websites/serializers.py

```diff
@@ -263,37 +263,62 @@ class Meta:
         ]
 
 
-class WebsiteCollectionSerializer(serializers.ModelSerializer):
+class WebsiteCollectionSerializer(
+    serializers.ModelSerializer, RequestUserSerializerMixin
+):
     """Serializer for WebsiteCollections"""
 
+    def create(self, validated_data):
+        """need to override to set owner"""
+        validated_data["owner"] = self.user_from_request()
+        with transaction.atomic():
+            collection = super().create(validated_data)
+        return collection
+
+    def update(self, instance, validated_data):
+        """gotta pop out owner if it's there"""
+        validated_data.pop("owner", None)
+        with transaction.atomic():
+            collection = super().update(instance, validated_data)
+        return collection
+
     class Meta:
         model = WebsiteCollection
         fields = ["title", "description"]
+        extra_kwargs = {"owner": {"write_only": True}}
 
 
 class WebsiteCollectionItemSerializer(serializers.ModelSerializer):
     """Serializer for WebsiteCollectionItems"""
 
     class Meta:
         model = WebsiteCollectionItem
-        fields = ["position", "website_collection", "website"]
+        read_only_fields = ["website_collection"]
```

Comment:
ah good idea! I wanted to include the `website_collection` sort of for the same reason, so that the `Item` records would always have all the info we need right there for making requests to the API.

Reply:
Probably should do the same for `collection` as well (http://localhost:8043/api/collections/)